README.md

```diff
@@ -1,39 +1,82 @@
+- **They work with or without Propel's APIs.** You can use them by themselves in static mode or with the Propel API in connected mode.
 
-Another advantage of the Propel UI Kit is that users can apply their own styles and custom error messages, giving them complete control over the look and feel of their data visualizations.
+## Key features
 
-## Getting Started
+All components share the same set of key features:
 
-To leverage the full power of Propel, you'll need a Propel account and a metric to work with. If you don't have an account yet, you can follow the instructions in [this article](https://propeldata.com/docs/metrics) to get started.
+- **They are responsive.** Components are responsive out of the box. They render well on desktops and laptops as well as phones and tablets.
 
-Once you have your account set up, you'll need to provide your account's access token to use Propel APIs. You can authenticate and generate an access token by following the steps outlined in [this article](https://propeldata.com/docs/api/authentication#step-2-generate-an-access-token).
+- **They handle loading and error states for you.** No need to sweat the details of handling the different states. We've done that for you.
 
-With your metric and access token in hand, it's time to start using Propel UI Kit. Currently, the UI Kit is available for React only and supports the following metric components:
+- **They fetch the data for you.** Components can automatically fetch the data from the Propel API when used in connected mode.
+
+Individual components have other key features that we highlight on the component documentation pages.
+
+## Components
+
+Check out the different components and their documentation:
 
 - [Time Series](https://github.com/propeldata/ui-kit/tree/main/packages/react/time-series)
 - [Counter](https://github.com/propeldata/ui-kit/tree/main/packages/react/counter)
 - [Leaderboard](https://github.com/propeldata/ui-kit/tree/main/packages/react/leaderboard)
 
-Now let's understand how you can integrate Propel UI Kit with your front-end application.
+Please [open a feature request](https://github.com/propeldata/ui-kit/issues) on GitHub for other components you'd like to have. Also, PRs are welcome 😊.
+
+## Getting Started with Propel
+
+To leverage the full power of Propel UI Kit + Propel's API, you'll need a Propel Account. If you don't have a Propel Account yet, you can [try Propel for free](https://propeldata.com/get-started).
+
+You can learn more about Propel by reading our [API docs](https://propeldata.com/docs).
+
+Once you have your account set up, you'll need to provide your Account's access token to use Propel APIs. You can authenticate and generate an access token by following the steps outlined in [the Propel GraphQL Authentication guide](https://propeldata.com/docs/api/authentication#step-2-generate-an-access-token).
```

Comment:
Nit: casing. Also, shouldn't we mention the Propel Application's access token, rather than the account's access token?

But also, this isn't really the right way to introduce authentication here, is it? As a customer of the UI Kit, I need to be thinking about how to 1. generate access tokens on demand, and 2. how to store them up until their expiration time securely in the browser.

For 1., if we're using Next.js — which React officially recommends — there are [API routes](https://nextjs.org/docs/api-routes/introduction) that can be used (i.e., set up an API route to return your Propel access token, applying the appropriate auth mechanisms). Or, maybe there is something else Next.js provides that is more appropriate? Need to research…

For 2., UI Kit should provide this as a feature. E.g., there should be some component, library function, etc., that we provide which customers can securely store and retrieve access tokens with. Perhaps the components should integrate with this, too (like context?). Need to research…

```
Once you have your account set up, you'll need to provide your account's access token to use Propel APIs. You can authenticate and generate an access token by following the steps outlined in [the Propel GraphQL Authentication guide](https://propeldata.com/docs/api/authentication#step-2-generate-an-access-token).
```